CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+
+* Enhancement - Add clear info message when chunks are removed because `storage.total_limit_size` is reached [fluent-bit:6719](https://github.com/fluent/fluent-bit/pull/6719)
```

Comment:
Supports this key enhancement to our oomkill guide: https://github.com/aws-samples/amazon-ecs-firelens-examples/pull/110

Reply:
Great! This sounds good.